### PR TITLE
Add no_codec condition to the doc and bump version

### PR DESCRIPTION
**Pull request by karenzone**
Added `no_codec` tag to doc and removed comment `//Verify default codec`
Included fix for #40 to add SNMP versions and protocols

### CODE REVIEW

**Comment by karenzone** [comment]
@colinsurprenant   I'd like to get your eyes on this.  Note that I'll need to get all the plumbing in place to support conditional coding before we can merge. Thanks!

**Comment by karenzone** [comment]
@colinsurprenant @acchen97  I added fix for #40 to this PR.

File: docs/index.asciidoc [hunk 1]
```diff
@@ -26,6 +24,8 @@ include::{include_path}/plugin_header.asciidoc[]
 The SNMP input polls network devices using Simple Network Management Protocol (SNMP)
 to gather information related to the current state of the devices operation.
 
+The SNMP input plugin supports SNMP v1, v2c, and v3 over UDP and TCP transport protocols.
```
Comment (acchen97) [inline docs/index.asciidoc]: Thanks @karenzone :+1:

**Comment by karenzone** [comment]
@colinsurprenant   This passed CI, but I got an error when I tried to publish the gem.

**Pull request closed without merging**